### PR TITLE
Feature xy

### PR DESCRIPTION
**Pull request by hvoelzer**
My new feature does some new astounding things ...

### CODE REVIEW

File: src/main/java/com/example/bookexample/service/BookServiceImpl.java
Comment (mafranco2000) [inline src/main/java/com/example/bookexample/service/BookServiceImpl.java:1]: All good

File: src/main/java/com/example/bookexample/service/BookServiceImpl.java [hunk 1]
```diff
@@ -12,9 +12,9 @@
 @Service // or @Service("BookService")
 public class BookServiceImpl implements BookService {
 
-    private BookRepository bookRepository;
+    private BookRepository bookRepository; // the underlying JPA repository
```
Comment (mafranco2000) [inline src/main/java/com/example/bookexample/service/BookServiceImpl.java:15]: Fine

**Review by mafranco2000 — APPROVED** [review]
Pls go ahead

File: src/main/java/com/example/bookexample/service/BookServiceImpl.java
Comment (LisZimmermann) [reply src/main/java/com/example/bookexample/service/BookServiceImpl.java:1]: This is exactly what we needed <3 Thanks for the effort and please go ahead!

**Review by LisZimmermann — APPROVED** [review]
Great, this is exactly what we needed! All fine IMO.

**Pull request merged**